api/discord/controllers/discord.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const axios = require('axios');
+
+module.exports = {
+  async discordAuth(ctx){
+    const { code } = ctx.query;
+    if (code) {
```

Comment:
While this is a 100% correct way to do it, you usually see people making if return statements instead of a big if statement like that in js code. Instead of the way you're checking if the code exits you can do.
```js
if(!code) return ctx.send({ message: 'There isn't a code in the url' }, 500);
```
Here's a [link](https://medium.com/@rachelchervin/if-else-vs-if-return-using-implicit-returns-in-javascript-and-react-bbac63656e1b) that explains what I'm talking about.